docs/source/devguide/docbuild.rst

```diff
@@ -35,21 +36,9 @@ Building Documentation as a PDF
 -------------------------------
 
 To build PDF documentation, you will need to have a LaTeX distribution installed
-on your computer as well as Inkscape_, which is used to convert .svg files to
-.pdf files. Inkscape can be installed in a Debian-derivative with:
-
-.. code-block:: sh
-
-    sudo apt-get install inkscape
-
-One the pre-requisites are installed, simply go to the ``docs`` directory and
-run:
+on your computer. Once you have a LaTeX distribution installed, simply go to the
+``docs`` directory and run:
 
 .. code-block:: sh
 
      make latexpdf
-
-.. _Sphinx: http://sphinx-doc.org
-.. _sphinxcontrib-tikz: https://bitbucket.org/philexander/tikz
```

Comment:
Nope, there are two TeX files that need to be converted, but we also just store the PNGs directly so that someone building the docs doesn't need to have that package,